editors/intellij/src/main/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,44 @@
+<!-- Plugin Configuration File. Read more: https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html -->
+<idea-plugin>
+    <id>com.github.biomejs.intellijbiome</id>
+    <name>Biome LSP</name>
+    <vendor>biomejs</vendor>
+
```

Comment:
After i remove changelog plugin from gradle, that description was extracted from README.md

What do you think? keep it manually or rollback the changelog plugin?

Reply:
> What do you think? keep it manually or rollback the changelog plugin?

I don't have any strong opinions. It may be better to write manually without using plugins since there are few people on the team who know Kotlin.